test/overlook-testing.js

```diff
@@ -2,11 +2,113 @@ import chai from 'chai';
 import { customerList } from './sample-data/sample-customer-list.js';
 import { roomList } from './sample-data/sample-room-list.js';
 import { bookingList } from './sample-data/sample-booking-list.js';
+import { checkUsername, getUserBookings, getTotalSpent  } from '../src/booking-utils.js';
 
 const expect = chai.expect;
 
-describe('See if the tests are running', function() {
-  it('should return true', function() {
-    expect(true).to.equal(true);
+describe('Get a customer\'s name from their username', () => {
```

Comment:
Actually, this does not work. Although I found an article saying you can make a conditional work by wrapping it in an ES5 function it didn't work as intended. You have to use a tertiary operator to add the conditional in the string interpolation. Forgot to do it in this line since I was not using this function yet. Check out the `populateBookings` function.

Reply:
True, but I had already added the single quotes and didn't want to go back and change them for backticks